### ISSUE TITLE
What unit is used for propagation distances?

### ISSUE THREAD

**Issue by rgov** [manoharan-lab/holopy#274]
The docs for `propagate()` say:

> *d* (float or list of floats) – Distance to propagate, in meters, or desired schema.

but the [tutorial](https://holopy.readthedocs.io/en/master/users/recon_tutorial.html) says:

> Next, we use numpy’s linspace to define a set of distances between the image plane and the reconstruction plane at 2-micron intervals to propagate our image to.

The code given generates this array of distances:

```python
array([ 0.,  2.,  4.,  6.,  8., 10., 12., 14., 16., 18., 20.])
```

**Comment by barkls** [comment]
In general, HoloPy is agnostic to units. In this case units are determined
by those used to define image spacing and illumination wavelength. the
docstring shouldn't mention metres.

**Comment by briandleahy** [comment]
Fixed by #291
